packages/types/package.json

```diff
@@ -28,7 +28,8 @@
     "@types/terser-webpack-plugin": "4.2.1",
     "@types/webpack": "^4.41.35",
     "@types/webpack-bundle-analyzer": "3.9.5",
-    "@types/webpack-hot-middleware": "2.25.5"
+    "@types/webpack-hot-middleware": "2.25.5",
+    "sass": "1.69.5"
```

Comment:
This change will most likely not be merged so my comment is not important but I don't want Nuxt pinning this to specific version as then it will cause duplicated packages and maybe even type errors when I use newer version in my project.